content/en/blog/_posts/2023-04-11-kubernetes-1.27-blog.md

```diff
@@ -0,0 +1,223 @@
+If enabled, the kubelet will use the `RuntimeDefault` seccomp profile by default, which is defined by the container runtime, instead of using the `Unconfined` (seccomp disabled) mode. The default profiles aim to provide a strong set of security defaults while preserving the functionality of the workload. It is possible that the default profiles differ between container runtimes and their release versions.
+
+You can find more detailed information about a possible upgrade and downgrade strategy in the related Kubernetes Enhancement Proposal (KEP): [Enable seccomp by default](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2413-seccomp-by-default).
+
+## Mutable scheduling directives for Jobs graduates to GA
+
+This was introduced in v1.22 and started as a beta level, now it's stable. In most cases a parallel job will want the pods to run with constraints, like all in the same zone, or all either on GPU model x or y but not a mix of both. The `suspend` field is the first step towards achieving those semantics. `suspend` allows a custom queue controller to decide when a job should start. However, once a job is unsuspended, a custom queue controller has no influence on where the pods of a job will actually land.
+
+This feature allows updating a Job's scheduling directives before it starts, which gives custom queue controllers
+the ability to influence pod placement while at the same time offloading actual pod-to-node assignment to
+kube-scheduler. This is allowed only for suspended Jobs that have never been unsuspended before.
+The fields in a Job's pod template that can be updated are node affinity, node selector, tolerations, labels
+and annotations and [scheduling gates](/docs/concepts/scheduling-eviction/pod-scheduling-readiness/).
+Find more details in the KEP:
+[Allow updating scheduling directives of jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2926-job-mutable-scheduling-directives).
+
+## DownwardAPIHugePages graduates to stable 
+
+In Kubernetes v1.20, support for `requests.hugepages-<pagesize>` and `limits.hugepages-<pagesize>` was added
+to the [downward API](/docs/concepts/workloads/pods/downward-api/) to be consistent with other resources like cpu, memory, and ephemeral storage.
+This feature graduates to stable in this release. You can find more details in the KEP:
+[Downward API HugePages](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2053-downward-api-hugepages).
+
+## Pod Scheduling Readiness goes to beta 
+
+Pods were considered ready for scheduling once created. Kubernetes scheduler does its due diligence to find nodes to place all pending Pods. However, in a real-world case, some Pods may stay in a _missing-essential-resources_ state for a long period. These Pods actually churn the scheduler (and downstream integrators like Cluster Autoscaler) in an unnecessary manner.
```

Comment:
Could you try to rephrase this sentence: `Pods were considered ready for scheduling once created.` The tense and meaning aren't clear to me.

Reply:
How about:
Upon creation, Pods are ready for scheduling.